pycobertura/reporters.py

```diff
@@ -274,6 +287,31 @@ def generate(self):
         return colored_json_string
 
 
+class YamlReporterDelta(DeltaReporter):
+    not_available = None
+
+    def generate(self):
+        lines = self.get_report_lines()
+
+        if self.show_source:
+            missed_lines_colored = [
+                self.color_number(line) for line in lines["Missing"]
+            ]
+            lines["Missing"] = missed_lines_colored
+
+        rows = {k: v[:-1] for k, v in lines.items()}
+        footer = {k: v[-1] for k, v in lines.items() if k != "Missing"}
+        # need to write to a buffer as yml packages are using a streaming interface
+        buf = io.BytesIO()
+        yaml.YAML().dump({"total": footer, "files": rows}, buf)
+        # need to replace \e escape sequence with \x1b,
+        # because only the latter is supported, see also
+        # the Python docs for supported formats:
+        # https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals
```

Comment:
I guess I still don't fully understand what is not supported. Is the escape sequence from the colored output? Then aren't those bytes meant to be interpreted by the terminal rather than Python itself?

Reply:
The escape sequence does not get printed in color, but is printed as is, whereas using \x1b color is printed using Python's print statement.
The guy here gives an example: 
https://stackoverflow.com/a/28934977/9698518